packages/stark-ui/src/util/form/form.util.ts

```diff
@@ -97,7 +97,7 @@ export class StarkFormUtil {
 	 * (calling `setFormControlState` function).
 	 * @param formGroup - Angular form group object
 	 */
-	public static isFormGroupValid(formGroup: FormGroup): boolean {
+	public static isUntypedFormGroupValid(formGroup: UntypedFormGroup): boolean {
```

Comment:
Could you rename this back to `isFormGroupValid` ?

Reply:
done